tests/main/snap-userd-desktop-app-autostart/task.yaml

```diff
@@ -1,5 +1,8 @@
 summary: Check that snap userd can autostart user session applications
 
+# ubuntu-14.04: too old journalctl
+systems: [-ubuntu-14.04-*]
```

Comment:
I think it would be better to only skip journalctl check on ubuntu-14.04 and test the rest of the functionality.

Reply:
Updated